integrations/instructor_embedders/pyproject.toml

```diff
@@ -30,6 +30,7 @@ dependencies = [
   # Commenting some of them to not interfere with the dependencies of Haystack.
   #"transformers==4.20.0",
   "datasets>=2.2.0",
+  "huggingface_hub<0.26.0",
```

Comment:
Unrelated. https://github.com/xlang-ai/instructor-embedding is practically abandoned and the release of `huggingface_hub` broke it.